windows/installer-source/KolibriSetupScript.iss

```diff
@@ -326,6 +328,14 @@ begin
         'System\CurrentControlSet\Control\Session Manager\Environment',
         'KOLIBRI_SCRIPT_DIR'
     )
+
+    KolibriHomePath := GetEnv('HOMEPATH') + '\.kolibri';
+    RegWriteStringValue(
+        HKLM,
+        'System\CurrentControlSet\Control\Session Manager\Environment',
+        'KOLIBRI_HOME',
+        KolibriHomePath
+    );

```

Comment:
I don't think we want to set `KOLIBRI_HOME` during installation. This should be fully managed by the end-user

Reply:
I think, it's important to have a default `KOLIBRI_HOME` during the installation. We just need to check if that environment exist during the installation to preserved the user config, If it doesn't exist let the installer setup set it.